src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+  /* margin: 0; */
+}
+
+h1 {
```

Comment:
add classes title, subtitle, paragraph, link and style by them, not by tag names